clang/test/Modules/gmodules-nodebug.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 -std=c++23 -x c++-header -emit-pch -fmodule-format=obj \
+// RUN:   -o %t.pch %s \
+// RUN:   -mllvm -debug-only=pchcontainer &>%t-pch.ll
+// RUN: cat %t-pch.ll | FileCheck %s
+
+template<class...>                     
```

Comment:
I just used exactly the template alias that was crashing in the libc++ case. I suppose it doesn't actually have to be a template